src/selectedtests/test_mappings/cli.py

```diff
@@ -0,0 +1,191 @@
+"""Cli entry point for the test-mappings command."""
+import json
+import logging
+import re
+import tempfile
+import click
+import structlog
+
+from datetime import datetime
+from typing import Dict
+from evergreen.api import CachedEvergreenApi
+from selectedtests.test_mappings.mappings import TestMappings
+from selectedtests.git_helper import init_repo
+
+LOGGER = structlog.get_logger(__name__)
+
+EXTERNAL_LIBRARIES = ["evergreen.api", "urllib3"]
+
+
+def _get_module_info(evg_api: CachedEvergreenApi, project: str, module_repo: str):
+    """
+    Fetch the module associated with an Evergreen project.
+
+    :param evg_api: An instance of the evg_api client
+    :param project: The name of the evergreen project to analyze.
+    :param module_name: Name of the module to analyze
+    :return: Information about the evergreen module
+    """
+    version_iterator = evg_api.versions_by_project(project)
+    recent_version = next(version_iterator)
+    modules = recent_version.get_manifest().modules
+    module = modules.get(module_repo)
+    return {
```

Comment:
It might be nice to, instead of transforming the return type here into a dict, return the module directly. It should have a type of ```ManifestModule```. If we then added that as a return type on the function, it would make the returned value and function a bit easier to use and reason about long term. 

As well, if we ever needed any of the other information about the module, such as the url or revision, we wouldn't have to update this function to add that info to the dict.